slashing-protection/src/test/java/tech/pegasys/web3signer/slashingprotection/DbSlashingProtectionTest.java

```diff
@@ -77,34 +78,43 @@ public void setup() {
   }
 
   @Test
-  public void blockCanSignWhenNoMatchForPublicKey() {
-    when(signedBlocksDao.findExistingBlock(any(), anyInt(), any())).thenReturn(Optional.empty());
+  public void blockCanSignWhenNoMatchForValidator() {
```

Comment:
These existing tests should probably moved into BlockValidatorTest and AttestationValidatorTest. Although we do need some tests showing that the order of rules is adhered too so do need something in the DbSlashingProtectionTest UTs.